tests/Unit/Plugin/Action/Order/ExportOrderActionTest.php

```diff
@@ -35,7 +35,7 @@
     /** @var \MyParcelNL\Pdk\Tests\Bootstrap\MockApiService $api */
     $api = Pdk::get(ApiServiceInterface::class);
     $api->getMock()
-        ->append(new ExamplePostOrdersResponse());
+        ->append(new ExamplePostOrdersResponse(), new ExamplePostOrderNotesResponse());
```

Comment:
test toevoegen

Reply:
Ik kan in deze test dit niet echt testen, dit is een response die nodig is voor de post order notes actie die binnen in de export order actie gebeurt. Er is wel een losse test voor de post order notes actie op zichzelf.